backend/api/Models/Concepts/Document/DocumentMetadataModel.cs

```diff
@@ -0,0 +1,21 @@
+namespace Pims.Api.Models.Concepts
+{
+    /// <summary>
+    /// DocumentMetadataModel class, provides a model to represent document metadata associated to entities.
+    /// </summary>
+    public class DocumentMetadataModel : BaseAppModel
```

Comment:
Pims does not have Document Metadata, instead, Mayan does. For that reason please move the model to the Mayan one on the Models folder as well as namespace

Reply:
Or if this is used by the frontend to pass down metadata maybe it should be renamed to something like DocumentMetadataUpdateModel

Reply:
Its used by frontend renaming it to DocumentMetadataUpdateModel